### PR TITLE
AT_01.06.007 | Header User menu My views link

### PR DESCRIPTION
**Pull request by LanaBondarenko**
https://trello.com/c/48UhdMqc/1200-at0106007-header-user-menu-my-views-link

### CODE REVIEW

**Comment by DariaMas7** [comment]
Attach please link to PR in your AT

File: cypress/e2e/headerMyViewMenuLinks.cy.js [hunk 1]
```diff
@@ -26,4 +26,11 @@ describe("US_01.06 Header My views menu link", () => {
         cy.url().should('includes', '/my-views/view/all/')
         cy.get(`[href="/user/${userName}/my-views/"]`).should('have.text', 'My Views')
     })
+
+    it('AT_01.06.007 | Header User menu My views link', function () {
+        cy.get('header .jenkins-menu-dropdown-chevron').realHover().click();
+        cy.get('a[href="/user/admin/my-views"]').click();
+        cy.url().should('includes', '/my-views');
+        cy.get('#breadcrumbs li').contains('My Views').should('be.visible');
```
Comment (DariaMas7) [inline cypress/e2e/headerMyViewMenuLinks.cy.js:34]: My Views should be moved to fixtures

Comment (LanaBondarenko) [reply cypress/e2e/headerMyViewMenuLinks.cy.js:34]: Daria, thanks for your comment. Do you think it makes sense to create a file for only one variable that is used only once in a test?

Comment (DariaMas7) [reply cypress/e2e/headerMyViewMenuLinks.cy.js:34]: From what I see, I mean how other students write tests, and based on comments from @Rom-T , yes. And there is no need to create a new json file, you may find an existing one the name of which relates to your US and simply add your key and value there -> import that file -> and use it in your code

Comment (Rom-T) [reply cypress/e2e/headerMyViewMenuLinks.cy.js:34]: You can take data for 33 and 34 rows from pages.json or homePage.json

**Review by Rom-T — APPROVED** [review]

**Review by TatianaBM — APPROVED** [review]
Lana, i would suggest to change css selector cy.get('a[href="/user/admin/my-views"]'). it looks like it contains user name admin . how about adding only last part of link a[href$="/my-views"]

**Pull request merged**